src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -61,62 +157,74 @@ public Command shooterIdle() {
         .withName("Idle shooter command");
   }
 
+  public Trigger shooterReady() {
+    return new Trigger(this::isShooterReady);
```

Comment:
Consider adding `.debounce(someNumber);` so there's a better chance the RPM is stable when we shoot

Reply:
Resolved in a99141b